src/data/repositories/consts/DiseaseOutbreakConstants.ts

```diff
@@ -33,3 +39,102 @@ export const DiseaseOutbreakCodes = {
     incidentManager: "RTSL_ZEB_TEA_ASSIGN_INCIDENT_MANAGER",
     notes: "RTSL_ZEB_TEA_NOTES",
 } as const;
+
+export type DiseaseOutbreakCode = GetValue<typeof DiseaseOutbreakCodes>;
+
+export type KeyCode = (typeof DiseaseOutbreakCodes)[keyof typeof DiseaseOutbreakCodes];
+
+export function isStringInDiseaseOutbreakCodes(code: string): code is KeyCode {
+    return Object.values(DiseaseOutbreakCodes).includes(code as KeyCode);
+}
+
+export function isHazardType(hazardType: string): hazardType is HazardType {
+    return [
+        "Biological:Animal",
+        "Biological:Human",
+        "Chemical",
+        "Environmental",
+        "Unknown",
+    ].includes(hazardType);
+}
```

Comment:
This is error-prone. You may write some string wrong and TS will follow along. The orthodox+single-truth way of doing is, in the entity:

```ts
const hazardTypes = [
    "Biological:Human",
    "Biological:Animal",
    "Chemical",
    "Environmental",
    "Unknown",
] as const;

export type HazardType = typeof hazardTypes[number];
```

You see, the other way around, you start from values and extract types from it. And now you can `hazardTypes` (useful for type validations, dropdown, and so on)